.github/workflows/bindings_python.yml

```diff
@@ -60,6 +60,10 @@ jobs:
       - uses: actions/checkout@v4
       - name: Setup Rust toolchain
         uses: ./.github/actions/setup
+      - name: Ruff Check
```

Comment:
The check results are the same on all platforms, so we don't need to check everywhere. How about checking it in the sdist job?